lint/src/main/kotlin/com/google/samples/apps/nowinandroid/lint/NiaIssueRegistry.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 The Android Open Source Project
+ * Copyright 2023 The Android Open Source Project
```

Comment:
nit: Copyright dates don't need to be updated. No need to change it back but might save you from some extra work for future changes.

Reply:
This was automatically update by my IDE I suppose...

```suggestion
 * Copyright 2022 The Android Open Source Project
```